moveit_ros/hybrid_planning/hybrid_planning_manager/hybrid_planning_manager_component/src/hybrid_planning_manager.cpp

```diff
@@ -45,7 +45,7 @@ namespace moveit::hybrid_planning
 using namespace std::chrono_literals;
 
 HybridPlanningManager::HybridPlanningManager(const rclcpp::NodeOptions& options)
```

Comment:
I see we never got rid of the ManagerManager name or the inheriting from rclcpp::Node.   :disappointed:

Reply:
We talked about it and decided it actually did make some sense in this case. I don't want to dig back through the different PR's to find the conversation right now, though

Reply:
I think we decided we were going to fix this.  I don't remember it being good for any reason.  I think I had a PR that fixed it too.  That being said I won't block this change for fixing this.

Reply:
I think it only made sense because we didn't have time to fix it at the time you were pushing to merge Hybrid Planner.  I don't remember any other reason but I do expect we will either fix this or document very clearly why this is necessary.

Reply:
https://github.com/ros-planning/moveit2/pull/488#discussion_r720463753

Reply:
What do you want to call it then? I'll do a quick find/replace.

Reply:
The link to the comment didn't work for some reason.  It just opens that giant PR.

I forgot I mentioned the SomethingManager.  I care much less about that than the inheriting from Node.  I don't have a better suggestion for the name at this point but I do know I don't like inheriting from the node object.

Reply:
https://github.com/ros-planning/moveit2/pull/488#discussion_r720408829

Reply:
Made an issue re. inheriting from Node here:  https://github.com/ros-planning/moveit2/issues/1013